### PR TITLE
Fix serial doctest + require passing tests on bors

### PR DESCRIPTION
**Pull request by eldruin**
This escaped CI somehow in #300.
Thanks to @adamgreig for his help in figuring this out.
Now passing the tests is also a requirement for bors.

### CODE REVIEW

**Review by therealprof — APPROVED** [review]
LGTM, thanks.

bors r+

**Comment by therealprof** [comment]
bors retry

File: .github/bors.toml [hunk 1]
```diff
@@ -6,5 +6,6 @@ status = [
   "ci-linux (stable, thumbv6m-none-eabi)",
   "ci-linux (stable, thumbv7m-none-eabi)",
   "ci-linux (1.35.0, x86_64-unknown-linux-gnu)",
+  "ci-linux-test (stable, x86_64-unknown-linux-gnu)",
```
Comment (adamgreig) [inline .github/bors.toml:9]:
```suggestion
  "ci-linux-test (stable)",
  "ci-linux-test (1.36.0, x86_64-unknown-linux-gnu)",
```

**Review by therealprof — APPROVED** [review]
Let's try this then. ;)

bors r+

**Pull request merged**